src/components/alert/alert-test.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import LinkTo from "@storybook/addon-links/react";
+import { action } from "@storybook/addon-actions";
+
+import Alert from ".";
+import Modal from "../modal";
+import Dialog from "../dialog";
+import Button from "../button";
+
+<Meta
+  title="Alert/Test"
+  parameters={{
+    info: { disable: true },
```

Comment:
You should add a parameter to disable knobs for this

Reply:
Good point, thank you. Changed.